modules/security-hardening-how.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-hardening.adoc
+
+[id="security-hardening-how_{context}"]
+
+= Choosing how to harden {op-system}
+
+Direct modification of {op-system} systems in {product-title} is discouraged.
+Instead, you should think of modifying systems in pools of nodes, such
+as worker nodes and master nodes. When a new node is needed, in
+non-bare metal installs, you can request a new node of the type
+you want and it will be created from an {op-system} image plus the
+modifications you created earlier.
+
+There are opportunities for modifying {op-system} before installation,
+during installation, and after the cluster is up and running.
+
+[[security-harden-before-installation]]
+== Hardening before installation
+
+For bare metal installations, you can add hardening features to
+{op-system} before beginning the {product-title} installation. For example,
+you can add kernel options when you boot the {op-system} installer
+to turn security features on or off, such as SELinux or various
+low-level settings, such as symmetric multithreading.
+
+Although bare metal {op-system} installations are more difficult,
+they offer the opportunity of getting operating system
+changes in place before starting the {product-title} installation. This can be important when you need to ensure that certain
+features, such as disk encryption or special networking settings, be
+set up at the earliest possible moment.
+
+[[security-harden-during-installation]]
+== Hardening during installation
+
+You can interrupt the `openshift-install` process and change
```

Comment:
Suggest "installation" or "OpenShift installation" instead of "openshift-install"

Reply:
Done.